ce/customer-service/voice-channel-ivr-bots.md

```diff
@@ -62,6 +71,17 @@ More information:
 
 - [Create topics and test your bot](https://go.microsoft.com/fwlink/?linkid=2062988)
 
+## Known issues
```

Comment:
@clmori We have moved away from documenting known issues in the docs site and instead the known issues are now available in a PDF that can be downloaded from the download center at https://go.microsoft.com/fwlink/p/?linkid=2165393. Therefore, I suggest that this section be removed. The PMs maintain this document; I can help you with updating it.

Reply:
@neeranelli these are known issues for Private Preview only. Does it still make sense to have it in that PDF? It might confuse the other customers who don't even know about the feature existence (it's not publicly released yet).

I'm happy to move if it makes sense though. FYI @kossrivastava

Reply:
@clmori @kossrivastava, One of the reasons we are not maintaining known issues in the docs site is because it is short-lived and doesn't justify the translation costs. Can this information be maintained on a support wiki?

Reply:
@neeranelli Is there a way to mark this page as "No translation"? It's already a NOINDEX,NOFOLLOW page, and I don't believe we need it to be translated.

In any case, I'm happy to add to a support wiki if a public one exists. Can you point me to it?